### DISCUSSION TITLE
Add script to increase swap size

### DISCUSSION THREAD

**Discussion by Serhioromano** [tteck/Proxmox#2477] [Ideas]
Default swap size in only 8Gb and it is full very fast. I increased it to 32Gb but that was an adventure. I immediately remembered you. that would be nice to have a script to increase a size of swap.

I was using this instruction
https://www.cyberciti.biz/faq/linux-add-a-swap-file-howto/

**Comment by tteck** [comment]
> Default swap size in only 8Gb

This swap?

![Screenshot 2024-02-12 7 03 19 AM](https://github.com/tteck/Proxmox/assets/21342670/f38946c1-f929-46b4-96f3-a83403f3744e)

**Reply by Serhioromano** [reply]
This one.

![2024-02-14_16-39-31](https://github.com/tteck/Proxmox/assets/650741/6b8813f8-6e9a-48a9-abf3-f9e2a4354cb2)
